.github/workflows/docker-based_tests.yml

```diff
@@ -35,9 +35,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-18.04]
-        go_version: ["go1.16.13", "go1.17.6", "go1.18beta1"]
+        go_version: ["go1.17.8", "go1.18"]
```

Comment:
why not use v1.17.x and v1.18.x, by the way? I see no reason to control the bugfix updates with precision, as they should never break any API or CLI command.

Reply:
We push version specific docker images to docker hub (as a manual step) and I'm not sure how well that would play with an "x-version" (see https://hub.docker.com/r/govim/govim/tags).

Regarding OS version, Paul had some comments in govim/govim#1130 where he had tested newer ubuntu and some tests failed.

I'll leave it as is and wrote govim/govim#1153 covering wildcard versions.

Reply:
SGTM! replied on the new issue.